Gemfile

```diff
@@ -119,7 +118,7 @@ gem 'autoprefixer-rails'
 gem 'slim'
 
 # Gravatar Image Tag
-gem 'gravatar_image_tag'
+gem 'gravatar_image_tag', github: 'Tinix/gravatar_image_tag'
```

Comment:
Original gravatar_image_tag gem is not updated anymore. It contains deprecated functions like URI.encode.

This fork addressed that. Though eventually we may want to move away totally from this (as I think gravatar_image is just an API after all so we maybe don't need our own gem for this)